docs/learning/tutorial.soy

```diff
@@ -41,6 +41,42 @@
 </blockquote>
 </p>
 
+<div class="toggler">
+  <table>
+    <tr>
+      <td><span><strong>Platform:</strong></span></td>
+      <td>
+        <a href="javascript:void(0);" class="button-android" onclick="display('platform', 'android')">Android</a>
+        <a href="javascript:void(0);" class="button-ios" onclick="display('platform', 'ios')">iOS</a>
+      </td>
+    </tr>
+    <tr>
+      <td><span><strong>Development OS:</strong></span></td>
+      <td>
+        <a href="javascript:void(0);" class="button-mac" onclick="display('os', 'mac')">macOS</a>
+        <a href="javascript:void(0);" class="button-linux" onclick="display('os', 'linux')">Linux</a>
+        <a href="javascript:void(0);" class="button-windows" onclick="display('os', 'windows')">Windows</a>
+      </td>
+    </tr>
+    <tr>
+      <td><span><strong>Language:</strong></span></td>
+      <td>
+        <a href="javascript:void(0);" class="button-java" onclick="display('language', 'java')">Java</a>
+        <a href="javascript:void(0);" class="button-kotlin" onclick="display('language', 'kotlin')">Kotlin</a>
+      </td>
+    </tr>
+  </table>
+</div>
+
+<p id="testanddebug">DEBUG</p>
+
+<span><block class="mac linux ios" /></span>
```

Comment:
Can probably remove these 2 blocks since we'll be disabling the option to even show them

Reply:
Removed